package.json

```
@@ -8,7 +8,8 @@
     "start": "next start",
     "lint": "next lint",
     "prepare": "husky install",
-    "release": "standard-version"
+    "release": "standard-version",
+    "test": "jest --watch"
```

I think it's better if you add another command just to run all of tests in CI environment. If you're only doing `jest`, then that command usually only runs the test for the files which have changed since the last commit (which may or may not be what you want, sometimes you have to press another key after the first test has finished running to run all of the tests).

```suggestion
    "test": "jest --watch"
    "test:ci": "jest --ci --collect-coverage --verbose"
```

---

Oh alright, after setup the unit test, I'll also setup the CI/CD, nice sugesstion